test/fixtures/schema.sql

```diff
@@ -305,7 +305,7 @@ SET search_path = postgrest, pg_catalog;
 
 CREATE TABLE auth (
     id character varying NOT NULL,
-    rolname name NOT NULL,
+    rolname name NOT NULL DEFAULT 'postgrest_test_author',
```

Comment:
Strangely when I remove the default value here the tests continue to pass. I was expecting a null violation. Does it do the same for you?